src/backends/backend/python/resources/startup.py

```diff
@@ -0,0 +1,163 @@
+import argparse
+import concurrent.futures as futures
+import importlib.util
+import sys
+import threading
+
+import numpy as np
+
+from python_host_pb2 import *
+from python_host_pb2_grpc import PythonInterpreterServicer, add_PythonInterpreterServicer_to_server
+import grpc
+
+_TYPE_MAPPING_DICTIONARY = {
+    # TRITONSERVER_TYPE_BOOL
+    1: np.bool,
+    # TRITONSERVER_TYPE_UINT8
+    2: np.uint8,
+    # TRITONSERVER_TYPE_UINT16
+    3: np.uint16,
+    # TRITONSERVER_TYPE_UINT32
+    4: np.uint32,
+    # TRITONSERVER_TYPE_UINT64
+    5: np.uint64,
+    # TRITONSERVER_TYPE_INT8
+    6: np.int8,
+    # TRITONSERVER_TYPE_INT16
+    7: np.int16,
+    # TRITONSERVER_TYPE_INT32
+    8: np.int32,
+    # TRITONSERVER_TYPE_INT64
+    9: np.int64,
+    # TRITONSERVER_TYPE_FP16
+    10: np.float16,
+    # TRITONSERVER_TYPE_FP32
+    11: np.float32,
+    # TRITONSERVER_TYPE_FP64
+    12: np.float64,
+    # TRITONSERVER_TYPE_BYTES
+    13: np.bytes_
+}
+
+
+def protobuf_to_numpy_type(data_type):
+    return _TYPE_MAPPING_DICTIONARY[data_type]
+
+
+def numpy_to_protobuf_type(data_type):
+    return {v: k for k, v in _TYPE_MAPPING_DICTIONARY.items()}[data_type]
```

Comment:
This is very inefficient. Implement like protobuf_to_numpy_type, or generate the table once at startup.